doc/psidk/src/specifications/app-architecture/supervisor.md

```diff
@@ -202,21 +274,22 @@ The supervisor listens for messages posted to its window messages with the follo
 
 ```json
 {
-    "message": {
-        "session": "number", // A number identifying the session/window on which the supervisor was instantiated
-        "type": "...", // Where the type is one of the supported supervisor message types
-        "payload": "..." // Contents depends on the message type
-    }
+  "message": {
+    "session": "number", // A number identifying the session/window on which the supervisor was instantiated
+    "type": "...", // Where the type is one of the supported supervisor message types
+    "payload": "..." // Contents depends on the message type
+  }
 }
 ```
 
 Supported supervisor message types:
-* `"Action"` - Used when requesting the supervisor add an action to the pending transaction
-* `"IAC"` - Used when calling a function defined in a plugin of another application
-* `"IACResponse"` - A response to a prior IAC
-* `"Event"` - A notification from the supervisor that an event to which your app subscribed has been emitted
-* `"TransactionReceipt"` - A response with the payload returned by the node to whom a transaction was submitted
-* `"ChangeHistory"` - A notification fired by a psibase app when its history changes, used for synchronizing the URL
+
+- `"Action"` - Used when requesting the supervisor add an action to the pending transaction
+- `"IAC"` - Used when calling a function defined in a plugin of another application
+- `"IACResponse"` - A response to a prior IAC
+- `"Event"` - A notification from the supervisor that an event to which your app subscribed has been emitted
+- `"TransactionReceipt"` - A response with the payload returned by the node to whom a transaction was submitted
+- `"ChangeHistory"` - A notification fired by a psibase app when its history changes, used for synchronizing the URL
```

Comment:
I know this hasn't been reviewed yet, but just FYI `ChangeHistory` will definitely not be needed anymore.

Reply:
do you want me to remove just this line? Or is there more editing to do to "remove" ChangeHistory?